core/forms.py

```diff
@@ -151,8 +151,9 @@ class AnnouncementForm(ModelForm):
         widget=forms.TextInput(attrs={'class': 'form-control rounded'}))
     target = forms.URLField(required=False, label='Target',
         widget=forms.URLInput(attrs={'class': 'form-control rounded'}))
-    body = forms.CharField(max_length=280, label='Body',
+    body = forms.CharField(max_length=500, label='Body',
         widget=forms.Textarea(attrs={'class': 'form-control rounded'}))
+    question = forms.BooleanField(required=False)
```

Comment:
This is nitpicking but can you make this variable name more descriptive? I think it's the send email Boolean but idk